README.md

```diff
@@ -63,6 +63,20 @@ you we created a parser that is local. You can open it by copying the path /
 browser then upload the claim.json file of that run.
 ![overview](docs/assets/images/htmlpage.png)
 
+## Upload a previous feedback to the html parser page
+
+A feature on the HTML parser page that we did to allow users to upload a previous
+feedback that they have to the parser page, they have an option to download the
+feedback then to upload it for the next time from the HTML page another option users
+can have that feedback file in the path #your-code-path/cnf-certification-test/script/
+and it need to be named as `feedback.js` users need to convert the downloaded
+`feedback.json` to `feedback.js` by running the command:
+`./tnf generate catalog feedbackjs -f path-to-feedbackjson/feedback.json `
```

Comment:
Please update this example. This command is not under "generate catalog" as per your last changes. Also, you should include the new "-o" flag in the example

Reply:
i did that

Reply:
Oh, my fault. Sorry.